content/en/integrations/guide/aws-manual-setup.md

```diff
@@ -45,57 +49,57 @@ Use this guide to manually set up the Datadog [AWS Integration][1].
 {{< tabs >}}
 {{% tab "Role delegation" %}}
 
-Setting up the AWS integration manually involves creating an IAM policy and IAM role in your AWS account, and configuring the role with an AWS External ID generated in your Datadog account. This allows Datadog's AWS account to query AWS APIs on your behalf, and pull data into your Datadog account. The sections below detail the steps for creating each of these components, and then completing the setup in your Datadog account.
+To set up the AWS integration manually, create an IAM policy and IAM role in your AWS account, and configure the role with an AWS External ID generated in your Datadog account. This allows Datadog's AWS account to query AWS APIs on your behalf, and pull data into your Datadog account. The sections below detail the steps for creating each of these components, and then completing the setup in your Datadog account.
 
 ## Setup
 
 ### Generate an external ID
-Generate an External ID in the <a href="https://app.datadoghq.com/account/settings#integrations/amazon_web_services" target="_blank">AWS integration tile</a> on the integrations page. This is used in the trust policy of the AWS IAM role you create for Datadog.
-1. Select the `Configuration` tab, then `Role Delegation`.
-2. Click `Manually`. This creates an AWS External ID which is used for configuration of the AWS IAM role. For more information about the External ID, see the [IAM User Guide][1].
-3. Copy this value to your clipboard or notepad.  
-  **Note: Do not close the integration tile or the Datadog application page**, as this causes the external ID value to reset.
 
-### AWS IAM Policy for Datadog
+1. In the [AWS integration tile][6], click **Add AWS Account**, and then select **Manually**.
```

Comment:
```suggestion
1. In the [AWS Integration configuration page][6], click **Add AWS Account**, and then select **Manually**.
```
Should be new link: https://app.datadoghq.com/integrations/amazon-web-services

Reply:
Thanks, updated!